frontend/src/app/settings/core/announcement-settings/edit-announcement/edit-announcement/edit-announcement.component.html

```diff
@@ -0,0 +1,62 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<form [formGroup]="announcementForm" class="collab-card">
+  <mat-form-field class="w-full" appearance="fill">
+    <mat-label>Title</mat-label>
+    <input matInput formControlName="title" />
+  </mat-form-field>
+  <mat-form-field class="w-full" appearance="fill">
+    <mat-label>Description</mat-label>
+    <textarea matInput formControlName="message"></textarea>
+    @if (
+      announcementForm.get("message")?.getError("titleOrDescriptionAvailable")
+    ) {
+      <mat-error>Please enter a description or title!</mat-error>
+    }
+  </mat-form-field>
+
+  <div class="flex items-center">
+    <mat-form-field appearance="fill">
+      <mat-label>Level</mat-label>
+      <mat-select formControlName="level">
+        @for (
+          announcementLevel of announcementLevels();
+          track announcementLevel
+        ) {
+          <mat-option [value]="announcementLevel">
+            {{ announcementLevel }}
+          </mat-option>
+        }
+      </mat-select>
+      <mat-error>Please select a level!</mat-error>
+    </mat-form-field>
+    <mat-checkbox class="pb-[22px]" formControlName="dismissible">
```

Comment:
```suggestion
    <mat-checkbox formControlName="dismissible">
```

With subscriptsizing, flex does the job.